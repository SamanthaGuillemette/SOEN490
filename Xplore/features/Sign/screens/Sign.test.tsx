import renderer from "react-test-renderer";
import { render } from "@testing-library/react-native";
import { NavigationContainer } from "@react-navigation/native";
import { createNativeStackNavigator } from "@react-navigation/native-stack";
import Sign from "./Sign.screen";

// Create stack navigator
const Stack = createNativeStackNavigator();

// Wrap test component in navigation container
function SignWithNavigation() {
  return (
    <NavigationContainer>
      <Stack.Screen name="Sign" component={Sign} />
    </NavigationContainer>
  );
}

describe("Tesing for <Sign /> screen:", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<SignWithNavigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders correctly", () => {
    const container = render(<SignWithNavigation />);
    expect(container).toBeTruthy();
  });
});
