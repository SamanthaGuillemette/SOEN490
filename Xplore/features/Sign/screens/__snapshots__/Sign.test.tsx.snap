// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tesing for <Sign /> screen: matches snapshot 1`] = `null`;
